calblueprint/berkeley-student-cooperative
# Andrei/shift schedule view

# ✨ New in this PR ✨

## One liner: what did you do? 🛠
Gives Shift Schedule view of current house.  Each row displays information about a given shift.
Clicking on shift makes modal appear (AssignShiftCard), that allows Manager to assign user to given task.


## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 -[x]  display shifts that have been assigned in a list view
- [x]  show their status
- [x]  display their time windows
- [x]  create an onClick action for each listed shift (we will later plug in the action that pops open more information about the shift)
    - [x]  future: plug in  [[Shift View Card Component]

## How can the reviewer test your code? 👩‍💻
Use the dropdown menu to select different days and check that the information matches to the shifts in Firebase.  
Verified Shifts subcollection hasn't been set up yet, so you can change numVerified in line 150 of ShiftSchedule.tsx and see how status changes.


## Any bugs you encountered or still having trouble with? 🐛
Sometimes a slight delay with click a row and the modal displaying correctly.  In future, could try loading multiple modals???


## Resources 📔
_link online resources and related PRs_ 


🏚 cc: @gregoriiaaa
